apps/web/pages/settings/profile.tsx

```diff
@@ -179,6 +179,9 @@ function SettingsView(props: ComponentProps<typeof Settings> & { localeProp: str
     const enteredLanguage = selectedLanguage.value;
     const enteredTimeFormat = selectedTimeFormat.value;
 
+    // Write time format to localStorage if available
+    window.localStorage.setItem("timeOption.is24hClock", selectedTimeFormat.value === 12 ? "false" : "true");
```

Comment:
When changing the time format in the settings page also saves the selection to localStorage